.evergreen/run-tests.sh

```diff
@@ -141,11 +141,15 @@ if [ -n "$TEST_ENCRYPTION" ]; then
     # TODO: Test with 'pip install pymongocrypt'
     git clone https://github.com/mongodb/libmongocrypt.git libmongocrypt_git
     python -m pip install --prefer-binary -r .evergreen/test-encryption-requirements.txt
-    python -m pip install ./libmongocrypt_git/bindings/python
+    #python -m pip install ./libmongocrypt_git/bindings/python
+    python -m pip install pymongocrypt@git+ssh://git@github.com/blink1073/libmongocrypt.git@PYTHON-3053#subdirectory=bindings/python
     python -c "import pymongocrypt; print('pymongocrypt version: '+pymongocrypt.__version__)"
     python -c "import pymongocrypt; print('libmongocrypt version: '+pymongocrypt.libmongocrypt_version())"
     # PATH is updated by PREPARE_SHELL for access to mongocryptd.
 
+    export CSFLE_TLS_CLIENT_PEM="$DRIVERS_TOOLS/.evergreen/x509gen/client.pem"
+    export CSFLE_TLS_CA_PEM="$DRIVERS_TOOLS/.evergreen/x509gen/ca.pem"
```

Comment:
Yes but only if the overall test is using SSL, which some of the CSFLE tests are not.

Reply:
They are always the same certs even when testing non tls clusters.
`CLIENT_PEM = os.environ.get("CLIENT_PEM", os.path.join(CERT_PATH, "client.pem"))`
is the same file as:
`CSFLE_TLS_CLIENT_PEM="$DRIVERS_TOOLS/.evergreen/x509gen/client.pem"`

Reply:
Right the env vars are only set conditionally but the CLIENT_PEM/CA_PEM globals in `test/__init__.py` are always available. Can we use those instead of adding new env vars?

Reply:
Ah, right, fixed